components/SnackBar/SnackBar.jsx

```diff
@@ -93,7 +96,26 @@ const TextContainer = styled.strong`
   }
 `;
 
-const CloseIcon = styled(Button.Icon).attrs({
+const SkinIcon = styled(Icon)`
```

Comment:
![image](https://user-images.githubusercontent.com/16689908/67128042-f8224d80-f1d0-11e9-80fd-421373dd6e21.png)

I've aligned with our product designer and he think is better to remove the hover effect around the close button.

Reply:
i did talk with him and he said to keep in this way.
Alright?